include/field_map_change.h

```diff
@@ -13,23 +13,23 @@ void sub_02053494(FieldSystem *fieldSystem);
 void FieldSystem_SetLoadNewGameSpawnTask(FieldSystem *fieldSystem);
 void FieldSystem_SetLoadSavedGameMapTask(FieldSystem *fieldSystem);
 void FieldSystem_StartLoadMapFromErrorTask(FieldSystem *fieldSystem);
-void FieldSystem_StartChangeMapTask(FieldTask *param0, const Location *param1);
-void FieldTask_ChangeMapByLocation(FieldTask *param0, const Location *param1);
-void FieldTask_ChangeMapToLocation(FieldTask *param0, int param1, int param2, int param3, int param4, int param5);
-void FieldTask_StartMapChangeFull(FieldTask *param0, int param1, int param2, int param3, int param4, int param5);
+void FieldSystem_StartChangeMapTask(FieldTask *taskMan, const Location *param1);
```

Comment:
It's being normalized to `task` for `FieldTask`, because there's no "management" aspect to it. There is a light call-stack, but that doesn't totally qualify it as a "manager" to me.

Reply:
should I change all of them to `task` then? I can just immediately do it in other files as well, if this is something that you want to normalize across the whole repo